content/en/docs/tutorials/compose-migration/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating from compose workflows to Kubernetes
+weight: 60
+---
+
+## Disclaimer
+
+The [compose specification](https://github.com/compose-spec/compose-spec) is evolving, this documentation aims to cover the latest version of this compose specification.
```

Comment:
The version of the compose-spec seems unclear. The reference implementation seems to be [Docker Compose v2](https://github.com/docker/compose). It would be good to clarify this.